lib/pbkit/pbkit.c

```diff
@@ -20,8 +20,9 @@
 #include <hal/io.h>
 #include <xboxkrnl/xboxkrnl.h>
 #include <hal/debug.h>
+#ifdef DBG
 #include <stdbool.h>
-#include <assert.h>
+#endif
```

Comment:
`stdbool` is also used unconditionally in functions like `pb_set_color_format` and `pb_init`

Reply:
Changes to includes reverted. Fixed.